installer/install.sh

```diff
@@ -243,6 +243,15 @@ fi
 echo "ONIE_IMAGE_PART_SIZE=$demo_part_size"
 
 extra_cmdline_linux=%%EXTRA_CMDLINE_LINUX%%
+# Inherit the FIPS option, so not necessary to do another reboot after upgraded
+if grep -qE '\ssonic_fips=1' /proc/cmdline && echo " $extra_cmdline_linux" | grep -qEv '\ssonic_fips='; then
```

Comment:
>\s [](http://example.com/codeflow?start=13&length=2)

You are expecting there is a space. If the flag is the first one, it will not match.

Reply:
This is a general logic. ref: [https://stackoverflow.com/questions/993452/splitting-proc-cmdline-arguments-with-spaces](https://stackoverflow.com/questions/993452/splitting-proc-cmdline-arguments-with-spaces)

Reply:
@qiluo-msft , thanks for your comment, looks like \\b is good.

Reply:
Changed to use \\b